### ISSUE TITLE
Edit Image icon does nothing

### ISSUE THREAD

**Issue by spencern** [reactioncommerce/reaction#2242]
![image](https://cloud.githubusercontent.com/assets/1203639/25927547/e509dbea-35b3-11e7-9b7d-0103a6eec775.png)

### Expected behavior
Something should happen when I click the button

### Actual Behavior
Nothing happens.

### Steps to Reproduce the Behavior
1. Clone dev
2. Login.
3. Add and image.
4. Click the "edit image" button in either the main image or the sub image section.

### Versions
```
Node: 7.8.0
NPM: 4.5.0
Meteor Node: 4.8.2
Meteor NPM: 4.5.0
Reaction CLI: 0.8.1
Reaction: 1.1.1
Reaction branch: brent-revert-permissions-fix
Docker: 17.03.1-ce
```

This button disappears after *_publishing_* the image

**Comment by zenweasel** [comment]
@spencern Is this a product image or a store image? Or both?

**Comment by aaronjudd** [comment]
@rymorgan I don't know if the design changed, but I noticed this as well. those don't look very styled, and look broken generally to me...  are they correct?

**Comment by spencern** [comment]
@zenweasel I've noticed this only on product images. Can't say for sure that it's not happening on store images though.

**Comment by kieckhafer** [comment]
This isn't the case of a non-working button, this is the case of a bad icon / button / indicator for what it's purpose is supposed to be.

That icon is just an indication that the image has changed, and isn't yet published. That means it's either a newly added image, or a deleted image, and when the product is published, the changes will be made. That's why it goes away when you publish.

@rymorgan perhaps we should change this to be the same as the products on the grid, with the little blue indicator on the top right?

![reaction](https://user-images.githubusercontent.com/4482263/27312010-dbb05b84-5519-11e7-8ca4-0d34f08adad1.png)

**Comment by zenweasel** [comment]
@kieckhafer I feel like this is 100% what should happen and pretty sure we had discussed this before and agreed that's what we should do.

**Comment by rnandhakumar1995** [comment]
In file **imports/plugins/core/ui/client/components/media/mediaGallery.js** enable the state **"disableClick"** of **Dropzone** component at line _131_

**Comment by kieckhafer** [comment]
@rnandhakumar1995 disableClick is already set to true

**Comment by kieckhafer** [comment]
Closed by #2469
